controllers/profile/profile.go

```diff
@@ -38,7 +42,23 @@ const (
 
 	longWait = 1 * time.Minute
 
-	errGetProfile = "cannot get profile"
+	errGetProfile                = "cannot get profile"
+	errConfigMapNil              = "config map cannot be nil"
+	errConfigMapWithoutNamespace = "config map must have a namespace"
+	errConfigMapWithoutName      = "config map must have a name"
+	errSavingProfile             = "cannot save profile"
+	errCreatingOperatorDir       = "cannot create operator directory"
+
+	seccompOperatorSuffix string      = "operator"
+	filePermissionMode    os.FileMode = 0600
+
+	// MkdirAll won't create a directory if it does not have the execute bit.
+	// https://github.com/golang/go/issues/22323#issuecomment-340568811
+	dirPermissionMode os.FileMode = 0700
+)
+
+var (
+	kubeletSeccompRootPath string = "/var/lib/kubelet/seccomp"
```

Comment:
This can be `const` I guess.

Reply:
Initially I created it as a `const`, but then changed it to make it testable - allowing tests to fiddle file/folder permissions without needing to run the tests as root. 

Assuming we will initially only support the operator to run as a `DaemonSet` inside the cluster (as opposed to running directly into the host nodes), this becomes less of a link to the actual seccomp root path on the host, but rather what is the mapped path inside the container. So I was wondering whether is there a better way to tackle this - any suggestions?

Reply:
Okay let's leave it like it is for now. Later on we can put this value into a configuration struct to ensure easier testing.